docker-compose.yml

```diff
@@ -11,22 +11,24 @@ services:
       - CB_HOST
       - CB_USER
       - CB_PSWD
-    entrypoint: ["./wait-for-couchbase.sh", "./entrypoint.sh"]
+      - COLUMNS
+    entrypoint: ["./wait-for-couchbase.sh", "node", "index.js"]
```

Comment:
ooo, could be. I haven't tested with `docker up` but should do... Will add that to todo list, ta!

Reply:
yeah, not sure if tested already, but we want to make sure this will behave the same way with a different Couchbase Server setup (i.e it will still create the FTS index and perform checks).

For example:
 `CB_HOST=192.168.0.2 CB_USER=Administrator CB_PSWD=password docker-compose up backend frontend`

Reply:
yeah, I've tested manually, to make sure it still worked as env vars instead of parameters, was one of the fun rabbitholes I went down yesterday... but definitely will re-test...

Reply:
Yeah I just want to make sure it runs those checks regardless of Couchbase Server flavour, but other than that this is ace! 👍 

I'll test this out with Python and Java whooop!

Reply:
Oh, so this is interesting, on e.g. `docker-compose up backend frontend`, I'm sure I've used this myself, but...

> `docker-compose up SERVICE` automatically includes `SERVICE`’s dependencies. In the example below, docker-compose up web also creates and starts `db` and `redis`.
> https://docs.docker.com/compose/compose-file/compose-file-v3/

😱 I'll do more testing, but given what's *documented* might move to advising using `docker up` manually in these cases?

Reply:
huh, yeah
```
❯ docker-compose up backend
Docker Compose is now in the Docker CLI, try `docker compose up`

Starting couchbase-sandbox-7.0.0-beta ... done
Starting try-cb-api                   ... done
```

Reply:
fairly sure it didn't use to do this...

Reply:
Hmm, because we are using `depends_on` here, it will still wait for dependencies to start I guess. Not sure if there is much we can do in that respect, even if you try to point to a different Couchbase server host it may still download the image for the Couchbase server image defined in the docker-compose file, even if it doesn't need to run it.

Reply:
yeah. oddly it doesn't *stop* the db when you ctrl-c to stop it though.
I'll rewrite to use `docker`